### PR TITLE
Update googlesearch ebuild

### PR DESCRIPTION
**Pull request by sectroyer**
Pentoo ebuild for googlesearch is very outdated and that version doesn't even work. I modified it to emerge version 3.0.

### CODE REVIEW

File: dev-python/googlesearch/googlesearch-3.0.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (ZeroChaos-) [inline dev-python/googlesearch/googlesearch-3.0.0-r1.ebuild:4]: 8 if it works

File: dev-python/googlesearch/googlesearch-3.0.0-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
```
Comment (ZeroChaos-) [inline dev-python/googlesearch/googlesearch-3.0.0-r1.ebuild:1]: What year is it?

File: dev-python/googlesearch/googlesearch-3.0.0-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+PYTHON_COMPAT=( python3_{8..9} )
+EGO_PN=github.com/MarioVilas/googlesearch
+
+inherit distutils-r1
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/MarioVilas/googlesearch.git"
+	KEYWORDS=""
```
Comment (ZeroChaos-) [inline dev-python/googlesearch/googlesearch-3.0.0-r1.ebuild:14]: Remove line please

**Comment by sectroyer** [comment]
I have removed the those lines and if statement. Code is from 2020 but for me it worked. No idea if all features work etc. but version 2.0 (currently in overlay) doesn't work at all 😄

**Comment by sectroyer** [comment]
Is this one okay? There wasn't any ${PYTHON_DEPS} in original ebuild so I didn't know it was required 😄

File: dev-python/googlesearch/googlesearch-3.0.0-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+DESCRIPTION="Google search from Python"
+HOMEPAGE="https://github.com/MarioVilas/googlesearch"
+S="${WORKDIR}/${PN}-${EGIT_COMMIT}"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE=""
+
+RDEPEND="${PYTHON_DEPS} >=dev-python/beautifulsoup4-4.5.1[${PYTHON_USEDEP}]"
```
Comment (ZeroChaos-) [inline dev-python/googlesearch/googlesearch-3.0.0-r1.ebuild:23]: Deps go on separate lines

**Pull request merged**